lib/inets/src/inets_app/inets.erl

```diff
@@ -104,12 +106,14 @@ stop() ->
 %% Description: Stops a started service of the inets application or takes
 %% down a stand alone "service" gracefully.
 %%--------------------------------------------------------------------
-stop(stand_alone, Pid) ->
-    true = exit(Pid, shutdown),
-    ok;
-
 stop(Service, Pid) ->
-    call_service(Service, stop_service, Pid).
+    case Service of
+        stand_alone ->
+            true = exit(Pid, shutdown),
+            ok;
+        _ ->
+            call_service(Service, stop_service, Pid)
+    end.
```

Comment:
I do not understand the point of this change! It changes nothing semantically! It only mixes the handling of the stand_alone case with the normal case in to one clause which I think makes the code less clear.

Reply:
Yep, sorry. Before a rebase, both function clauses were sharing a function call, so putting them together avoids repetition of the same call in two different function clauses. Now that @Maria-12648430 pointed me to remove the call to `stop`, which was 100% correct, I will put this back as it was (as you suggest @IngelaAndin )
Thanks!